user/test/com/google/gwt/emultest/java11/util/function/PredicateTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 Google Inc.
```

Comment:
```suggestion
 * Copyright 2023 GwtProject contributors
```

Reply:
@zbynek As far as I know this change should only be done after https://github.com/gwtproject/gwt/issues/9823 has been merged. Also it is generally better to have one commit change it everywhere.

Reply:
Based on https://github.com/gwtproject/gwt/blob/main/user/test-super/com/google/gwt/dev/jjs/super/com/google/gwt/dev/jjs/test/Java9Test.java#L2 I thought the policy for new files already changed, but apparently not. Thanks for the link.